assets/js/theme/global/modal.js

```diff
@@ -196,12 +213,89 @@ export class Modal {
         this.$content.html('');
     }
 
+    setupFocusableElements(modalType) {
+        this.$preModalFocusedEl = $(document.activeElement);
+        
+        const $collection = focusableElements[modalType]();
+
+        this.findElementToFocus($collection);
+
+        const onTabbingWithArgs = (event) => this.onTabbing(event, modalType, this.findElementToFocus.bind(this));
```

Comment:
We can pass onTabbingWithArgs func directly in event listener, no need in separate creating